spec/fabricators/user_fabricator.rb

```diff
@@ -54,8 +54,7 @@
 
   after_create do |user|
     user.group_users << Fabricate(:group_user, user: user, group: Group[:moderators])
-    # HACK: Some plugins add the user to staff group already which breaks because of duplication
```

Comment:
Is there more details on why this is no longer required/relevant?

Reply:
I added the `if ...` because some plugins were fixing it. One such plugin was discourse-assign:

https://github.com/discourse/discourse-assign/pull/404

Now that it is finally fixed, we do not need the hack anymore.

Reply:
Thank you for clarifying 👍